modules/tinymce/src/core/main/ts/paste/ProcessFilters.ts

```diff
@@ -25,7 +25,7 @@ const preProcess = (editor: Editor, html: string): string => {
     });
   });
 
-  const fragment = parser.parse(html, { forced_root_block: false, isRootContent: true });
+  const fragment = parser.parse(html, { isRootContent: true });
```

Comment:
Just want to sanity check that this is correct. 

With the change made at https://github.com/tinymce/tinymce/pull/9451/files#diff-6f02c7b2bc90a7443c69cbcc401505fa683e4e4c09192fa3eb36629373bdfb4dR258 and not explicitly setting `forced_root_block: ''` it seems like the forced root block could either undefined or default to 'p'?

Maybe I am missing some other code somewhere that makes not specificing `forced_root_block` safe